apps/librepcb/controlpanel/controlpanel.cpp

```diff
@@ -411,14 +412,11 @@ void ControlPanel::projectEditorClosed() noexcept
 
 void ControlPanel::on_actionAbout_triggered()
 {
-    QMessageBox::about(this, tr("About LibrePCB"), QString(tr(
-        "<h1>About LibrePCB</h1>"
-        "<p>LibrePCB is a free & open source schematic/layout-editor.</p>"
-        "<p>Version: %1 (%2)</p>"
-        "<p>Please see <a href='http://librepcb.org/'>librepcb.org</a> for more information.</p>"
-        "You can find the project on GitHub:<br>"
-        "<a href='https://github.com/LibrePCB/LibrePCB'>https://github.com/LibrePCB/LibrePCB</a>"))
-        .arg(qApp->getAppVersion().toPrettyStr(3), qApp->getGitVersion()));
+    const Version& appVersion = qApp->getAppVersion();
+    const QString& gitVersion = qApp->getGitVersion();
+    const QString& buildDate = qApp->getBuildDate();
+    AboutDialog* aboutDialog = new AboutDialog(this, appVersion, gitVersion, buildDate);
```

Comment:
- Why passing version information to the about dialog? If the dialog is also used in the other main windows (schematic/board/library editors), this code needs to be copied each time.
- The AboutDialog can be created on the stack, no need to use heap for that.

Reply:
> Why passing version information to the about dialog?

I could not access `qApp` inside the dialog even when including the header file. Maybe you can try it? I probably just overlooked something.

Reply:
I guess the problem is that `AboutDialog` is not (yet) in the `librepcb` namespace.

Reply:
Ahh, that could be :)